Rocket.Chat.Shared/Controllers/Chat/ChatViewController.swift

```diff
@@ -452,7 +452,14 @@ public class ChatViewController: SLKTextViewController, AuthManagerInjected, Soc
                 let indexPaths = self.dataController.insert(objs)
                 collectionView.insertItems(at: indexPaths)
                 if self.messageCellStyle == .bubble {
-                    collectionView.reloadItems(at: indexPaths.filter { $0.row > 0 }.map { IndexPath(row: $0.row - 1, section: $0.section) }.filter { !indexPaths.contains($0) })
+                    collectionView.reloadItems(at: indexPaths
+                        // Only items not the first
+                        .filter { $0.row > 0 }
```

Comment:
You can find that we have to minus 1 to get its previous one. the first one obviously does not have a previous one.